Dockerfile

```
@@ -15,7 +15,8 @@ ENV BUILD_ENV=$BUILD_ENV \
     LC_ALL="en_US.UTF-8" \
     LANGUAGE="en_US:en" \
     NODE_ENV=$NODE_ENV \
-    NODE_VERSION=20
+    NODE_VERSION=20 \
+    PUPPETEER_EXECUTABLE_PATH="/usr/bin/google-chrome"
```

```suggestion
    CHROME_EXECUTABLE_PATH="/usr/bin/google-chrome"
```

I am a bit curious because it's google chrome executable file. But if this is the constant which used by the Puppeteer then it's fine to use this name.

---

That constant used by Puppeteer. You can refer to this [document](https://pptr.dev/api/puppeteer.configuration)